src/Mvc/Mvc.Core/src/ModelBinding/Binders/FloatModelBinder.cs

```diff
@@ -73,7 +72,7 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
                 }
                 else if (type == typeof(float))
                 {
-                    model = float.Parse(value, _supportedStyles, culture);
+                    model = float.Parse(value, _supportedStyles, valueProviderResult.Culture);
```

Comment:
Any functional reason for inlining this or just code clarity?

Reply:
See https://github.com/dotnet/aspnetcore/pull/24893#discussion_r470365417